src/xdocs/config_filters.xml

```diff
@@ -2296,6 +2296,14 @@ public class UserService {
               <td><code>null</code></td>
               <td>8.24</td>
             </tr>
+            <tr>
+              <td>influenceFormat</td>
+              <td>Specify negative/zero/positive value that defines the number of
+                  lines preceding/at/following the suppression comment.</td>
+              <td><a href="property_types.html#String">String</a></td>
+              <td><code>"0"</code></td>
+              <td>5.0</td>
```

Comment:
Put here 10.0 for now it is our next version.
This version define in what version this property appears.
So users that use old version should clearly see it